CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.0
+
+The is a major release
+
+* [TK-369](https://tickets.puppetlabs.com/browse/TK-369) Move Jetty dependency to 9.4.1
```

Comment:
I *think* jetty considers them to not be major releases, but it's hard to get at from their docs. I guess the java 7 drop is probably the biggest user facing change.